fe1-web/src/core/components/RoundIconButton.tsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { StyleSheet, TouchableOpacity, ViewStyle } from 'react-native';
+import { Icon } from 'react-native-elements';
+
+const styles = StyleSheet.create({
+  roundButton: {
+    height: 65,
+    width: 65,
+    shadowColor: 'gray',
+    shadowOffset: {
+      width: 2,
+      height: 2,
+    },
+    borderRadius: 30,
+    backgroundColor: '#2196F3',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
```

Comment:
In general we should try to avoid constants in styles as much as possible and refer to global variables instead. I am doing some work on a PR regarding this since it is the case all over the application 😅 But it would be cool if you could for example reference a color define in `core/styles/colors` as it will make UI changes much much easier :)

Reply:
I agree. This is cleaner and will be much easier to keep consistent.